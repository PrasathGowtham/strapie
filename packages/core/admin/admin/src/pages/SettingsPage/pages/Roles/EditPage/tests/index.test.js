/**
 *
 * Tests for EditPage
 *
 */

import React from 'react';
import { render } from '@testing-library/react';
import { IntlProvider } from 'react-intl';
import { Router, Switch, Route } from 'react-router-dom';
import { createMemoryHistory } from 'history';
import { lightTheme, darkTheme } from '@strapi/design-system';
import Theme from '../../../../../../components/Theme';
import ThemeToggleProvider from '../../../../../../components/ThemeToggleProvider';

import EditPage from '../index';

jest.mock('@strapi/helper-plugin', () => ({
  ...jest.requireActual('@strapi/helper-plugin'),
  useNotification: jest.fn(() => jest.fn()),
  useOverlayBlocker: jest.fn(() => ({ lockApp: jest.fn(), unlockApp: jest.fn() })),
  useTracking: jest.fn(() => ({ trackUsage: jest.fn() })),
}));

const makeApp = (history) => (
  <IntlProvider
    messages={{ 'Settings.roles.form.created': 'created' }}
    textComponent="span"
    locale="en"
    defaultLocale="en"
  >
    <ThemeToggleProvider themes={{ light: lightTheme, dark: darkTheme }}>
      <Theme>
        <Router history={history}>
          <Switch>
            <Route path="/settings/roles/:id">
              <EditPage />
            </Route>
          </Switch>
        </Router>
      </Theme>
    </ThemeToggleProvider>
  </IntlProvider>
);

describe('<EditPage />', () => {
  it('renders and matches the snapshot', () => {
    const history = createMemoryHistory();
    const App = makeApp(history);
    const { container } = render(App);

    history.push('/settings/roles/1');

    expect(container).toMatchSnapshot();
  });
});
