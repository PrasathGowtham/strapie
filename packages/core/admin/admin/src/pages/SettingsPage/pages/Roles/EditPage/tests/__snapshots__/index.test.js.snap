// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EditPage /> renders and matches the snapshot 1`] = `
.c37 {
  border: 0;
  -webkit-clip: rect(0 0 0 0);
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  width: 1px;
}

.c38 {
  -webkit-animation: gzYjWD 1s infinite linear;
  animation: gzYjWD 1s infinite linear;
  will-change: transform;
}

.c2 {
  background: #f6f6f9;
  padding-top: 24px;
  padding-right: 56px;
  padding-bottom: 40px;
  padding-left: 56px;
}

.c3 {
  padding-bottom: 8px;
}

.c5 {
  padding-right: 8px;
}

.c17 {
  padding-right: 56px;
  padding-left: 56px;
}

.c20 {
  background: #ffffff;
  padding: 24px;
  border-radius: 4px;
  box-shadow: 0px 1px 4px rgba(33,33,52,0.1);
}

.c9 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c10 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c18 {
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c35 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
}

.c8 {
  font-size: 0.875rem;
  line-height: 1.43;
  color: #4945ff;
}

.c11 {
  font-weight: 600;
  font-size: 2rem;
  line-height: 1.25;
  color: #32324d;
}

.c15 {
  font-size: 0.875rem;
  line-height: 1.43;
  font-weight: 600;
  line-height: 1.14;
  color: #32324d;
}

.c16 {
  font-size: 1rem;
  line-height: 1.5;
  color: #666687;
}

.c22 {
  font-size: 0.875rem;
  line-height: 1.43;
  font-weight: 600;
  color: #32324d;
}

.c23 {
  font-size: 0.75rem;
  line-height: 1.33;
  color: #8e8ea9;
}

.c25 {
  font-size: 0.75rem;
  line-height: 1.33;
  font-weight: 600;
  line-height: 1.14;
  color: #32324d;
}

.c29 {
  font-size: 0.75rem;
  line-height: 1.33;
  font-weight: 600;
  color: #32324d;
}

.c19 > * {
  margin-top: 0;
  margin-bottom: 0;
}

.c19 > * + * {
  margin-top: 24px;
}

.c21 > * {
  margin-top: 0;
  margin-bottom: 0;
}

.c21 > * + * {
  margin-top: 16px;
}

.c28 > * {
  margin-top: 0;
  margin-bottom: 0;
}

.c28 > * + * {
  margin-top: 4px;
}

.c12 > * {
  margin-left: 0;
  margin-right: 0;
}

.c12 > * + * {
  margin-left: 8px;
}

.c13 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  cursor: pointer;
  padding: 8px;
  border-radius: 4px;
  background: #ffffff;
  border: 1px solid #dcdce4;
  position: relative;
  outline: none;
}

.c13 svg {
  height: 12px;
  width: 12px;
}

.c13 svg > g,
.c13 svg path {
  fill: #ffffff;
}

.c13[aria-disabled='true'] {
  pointer-events: none;
}

.c13:after {
  -webkit-transition-property: all;
  transition-property: all;
  -webkit-transition-duration: 0.2s;
  transition-duration: 0.2s;
  border-radius: 8px;
  content: '';
  position: absolute;
  top: -4px;
  bottom: -4px;
  left: -4px;
  right: -4px;
  border: 2px solid transparent;
}

.c13:focus-visible {
  outline: none;
}

.c13:focus-visible:after {
  border-radius: 8px;
  content: '';
  position: absolute;
  top: -5px;
  bottom: -5px;
  left: -5px;
  right: -5px;
  border: 2px solid #4945ff;
}

.c14 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #4945ff;
  border: 1px solid #4945ff;
  height: 2.5rem;
  padding-left: 16px;
  padding-right: 16px;
}

.c14 .c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c14 .c7 {
  color: #ffffff;
}

.c14[aria-disabled='true'] {
  border: 1px solid #dcdce4;
  background: #eaeaef;
}

.c14[aria-disabled='true'] .c7 {
  color: #666687;
}

.c14[aria-disabled='true'] svg > g,.c14[aria-disabled='true'] svg path {
  fill: #666687;
}

.c14[aria-disabled='true']:active {
  border: 1px solid #dcdce4;
  background: #eaeaef;
}

.c14[aria-disabled='true']:active .c7 {
  color: #666687;
}

.c14[aria-disabled='true']:active svg > g,.c14[aria-disabled='true']:active svg path {
  fill: #666687;
}

.c14:hover {
  border: 1px solid #7b79ff;
  background: #7b79ff;
}

.c14:active {
  border: 1px solid #4945ff;
  background: #4945ff;
}

.c14 svg > g,
.c14 svg path {
  fill: #ffffff;
}

.c24 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #4945ff;
  border: 1px solid #4945ff;
  height: 2rem;
  padding-left: 16px;
  padding-right: 16px;
  border: 1px solid #d9d8ff;
  background: #f0f0ff;
}

.c24 .c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c24 .c7 {
  color: #ffffff;
}

.c24[aria-disabled='true'] {
  border: 1px solid #dcdce4;
  background: #eaeaef;
}

.c24[aria-disabled='true'] .c7 {
  color: #666687;
}

.c24[aria-disabled='true'] svg > g,.c24[aria-disabled='true'] svg path {
  fill: #666687;
}

.c24[aria-disabled='true']:active {
  border: 1px solid #dcdce4;
  background: #eaeaef;
}

.c24[aria-disabled='true']:active .c7 {
  color: #666687;
}

.c24[aria-disabled='true']:active svg > g,.c24[aria-disabled='true']:active svg path {
  fill: #666687;
}

.c24:hover {
  background-color: #ffffff;
}

.c24:active {
  background-color: #ffffff;
  border: 1px solid #4945ff;
}

.c24:active .c7 {
  color: #4945ff;
}

.c24:active svg > g,
.c24:active svg path {
  fill: #4945ff;
}

.c24 .c7 {
  color: #271fe0;
}

.c24 svg > g,
.c24 svg path {
  fill: #271fe0;
}

.c31 {
  border: none;
  border-radius: 4px;
  padding-bottom: 0.65625rem;
  padding-left: 16px;
  padding-right: 16px;
  padding-top: 0.65625rem;
  color: #32324d;
  font-weight: 400;
  font-size: 0.875rem;
  display: block;
  width: 100%;
  background: inherit;
}

.c31::-webkit-input-placeholder {
  color: #8e8ea9;
  opacity: 1;
}

.c31::-moz-placeholder {
  color: #8e8ea9;
  opacity: 1;
}

.c31:-ms-input-placeholder {
  color: #8e8ea9;
  opacity: 1;
}

.c31::placeholder {
  color: #8e8ea9;
  opacity: 1;
}

.c31[aria-disabled='true'] {
  color: inherit;
}

.c31:focus {
  outline: none;
  box-shadow: none;
}

.c30 {
  border: 1px solid #dcdce4;
  border-radius: 4px;
  background: #ffffff;
  outline: none;
  box-shadow: 0;
  -webkit-transition-property: border-color,box-shadow,fill;
  transition-property: border-color,box-shadow,fill;
  -webkit-transition-duration: 0.2s;
  transition-duration: 0.2s;
}

.c30:focus-within {
  border: 1px solid #4945ff;
  box-shadow: #4945ff 0px 0px 0px 2px;
}

.c33 {
  border: 1px solid #dcdce4;
  border-radius: 4px;
  padding-left: 16px;
  padding-right: 16px;
  padding-top: 12px;
  padding-bottom: 12px;
  background: #ffffff;
  outline: none;
  box-shadow: 0;
  -webkit-transition-property: border-color,box-shadow,fill;
  transition-property: border-color,box-shadow,fill;
  -webkit-transition-duration: 0.2s;
  transition-duration: 0.2s;
}

.c33:focus-within {
  border: 1px solid #4945ff;
  box-shadow: #4945ff 0px 0px 0px 2px;
}

.c34 {
  display: block;
  width: 100%;
  font-weight: 400;
  font-size: 0.875rem;
  border: none;
  color: #32324d;
  resize: none;
  background: inherit;
}

.c34::-webkit-input-placeholder {
  color: #8e8ea9;
  opacity: 1;
}

.c34::-moz-placeholder {
  color: #8e8ea9;
  opacity: 1;
}

.c34:-ms-input-placeholder {
  color: #8e8ea9;
  opacity: 1;
}

.c34::placeholder {
  color: #8e8ea9;
  opacity: 1;
}

.c34:focus-within {
  outline: none;
}

.c32 textarea {
  height: 5rem;
  line-height: 1.25rem;
}

.c32 textarea::-webkit-input-placeholder {
  font-weight: 400;
  font-size: 0.875rem;
  line-height: 1.43;
  color: #8e8ea9;
  opacity: 1;
}

.c32 textarea::-moz-placeholder {
  font-weight: 400;
  font-size: 0.875rem;
  line-height: 1.43;
  color: #8e8ea9;
  opacity: 1;
}

.c32 textarea:-ms-input-placeholder {
  font-weight: 400;
  font-size: 0.875rem;
  line-height: 1.43;
  color: #8e8ea9;
  opacity: 1;
}

.c32 textarea::placeholder {
  font-weight: 400;
  font-size: 0.875rem;
  line-height: 1.43;
  color: #8e8ea9;
  opacity: 1;
}

.c4 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  position: relative;
  outline: none;
}

.c4 svg path {
  fill: #4945ff;
}

.c4 svg {
  font-size: 0.625rem;
}

.c4:after {
  -webkit-transition-property: all;
  transition-property: all;
  -webkit-transition-duration: 0.2s;
  transition-duration: 0.2s;
  border-radius: 8px;
  content: '';
  position: absolute;
  top: -4px;
  bottom: -4px;
  left: -4px;
  right: -4px;
  border: 2px solid transparent;
}

.c4:focus-visible {
  outline: none;
}

.c4:focus-visible:after {
  border-radius: 8px;
  content: '';
  position: absolute;
  top: -5px;
  bottom: -5px;
  left: -5px;
  right: -5px;
  border: 2px solid #4945ff;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c36 {
  height: 100vh;
}

.c26 {
  display: grid;
  grid-template-columns: repeat(12,1fr);
  gap: 16px;
}

.c27 {
  grid-column: span 6;
  max-width: 100%;
}

.c0:focus-visible {
  outline: none;
}

@media (max-width:68.75rem) {
  .c27 {
    grid-column: span;
  }
}

@media (max-width:34.375rem) {
  .c27 {
    grid-column: span;
  }
}

<div>
  <main
    aria-labelledby="main-content-title"
    class="c0"
    id="main-content"
    tabindex="-1"
  >
    <form>
      <div>
        <div
          class="c1 c2"
          data-strapi-header="true"
        >
          <div
            class="c1 c3"
          >
            <a
              aria-current="page"
              class="c4 active"
              href="/settings/roles"
            >
              <span
                aria-hidden="true"
                class="c1 c5 c6"
              >
                <svg
                  fill="none"
                  height="1em"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M24 13.3a.2.2 0 01-.2.2H5.74l8.239 8.239a.2.2 0 010 .282L12.14 23.86a.2.2 0 01-.282 0L.14 12.14a.2.2 0 010-.282L11.86.14a.2.2 0 01.282 0L13.98 1.98a.2.2 0 010 .282L5.74 10.5H23.8c.11 0 .2.09.2.2v2.6z"
                    fill="#212134"
                  />
                </svg>
              </span>
              <span
                class="c7 c8"
              >
                Back
              </span>
            </a>
          </div>
          <div
            class="c1 c9"
          >
            <div
              class="c1 c10"
            >
              <h1
                class="c7 c11"
              >
                Edit a role
              </h1>
            </div>
            <div
              class="c1 c10 c12"
              spacing="2"
            >
              <button
                aria-disabled="false"
                class="c13 c14"
                type="button"
              >
                <span
                  class="c7 c15"
                >
                  Save
                </span>
              </button>
            </div>
          </div>
          <p
            class="c7 c16"
          >
            Define the rights given to the role
          </p>
        </div>
      </div>
      <div
        class="c1 c17"
      >
        <div
          class="c1 c18 c19"
          spacing="6"
        >
          <div
            class="c1 c20"
          >
            <div
              class="c1 c18 c21"
              spacing="4"
            >
              <div
                class="c1 c9"
              >
                <div
                  class="c1 "
                >
                  <div
                    class="c1 "
                  >
                    <span
                      class="c7 c22"
                    />
                  </div>
                  <div
                    class="c1 "
                  >
                    <span
                      class="c7 c23"
                    />
                  </div>
                </div>
                <button
                  aria-disabled="true"
                  class="c13 c24"
                  disabled=""
                  type="button"
                >
                  <span
                    class="c7 c25"
                  >
                    NaN users with this role
                  </span>
                </button>
              </div>
              <div
                class="c1 c26"
              >
                <div
                  class="c27"
                >
                  <div
                    class="c1 "
                  >
                    <div>
                      <div>
                        <div
                          class="c1 c18 c28"
                          spacing="1"
                        >
                          <label
                            class="c7 c29"
                            for="textinput-1"
                          >
                            <div
                              class="c1 c10"
                            >
                              Name
                            </div>
                          </label>
                          <div
                            class="c1 c9 c30"
                          >
                            <input
                              aria-disabled="false"
                              aria-invalid="false"
                              class="c31"
                              id="textinput-1"
                              name="name"
                              value=""
                            />
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="c27"
                >
                  <div
                    class="c1 "
                  >
                    <div
                      class="c32"
                    >
                      <div>
                        <div
                          class="c1 c18 c28"
                          spacing="1"
                        >
                          <div
                            class="c1 c10"
                          >
                            <label
                              class="c7 c29"
                              for="textarea-2"
                            >
                              <div
                                class="c1 c10"
                              >
                                Description
                              </div>
                            </label>
                          </div>
                          <div
                            class="c33"
                          >
                            <textarea
                              aria-invalid="false"
                              class="c34"
                              id="textarea-2"
                              name="description"
                            />
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="c1 c20"
          >
            <div
              class="c1 c35 c36"
              data-testid="loader"
            >
              <div
                aria-live="assertive"
                role="alert"
              >
                <div
                  class="c37"
                >
                  Loading content.
                </div>
                <img
                  aria-hidden="true"
                  class="c38"
                  src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNjMiIGhlaWdodD0iNjMiIHZpZXdCb3g9IjAgMCA2MyA2MyIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZD0iTTQyLjU1NjMgMTEuOTgxNkMzOS40ODQgMTAuMzA3MSAzNS44NTc1IDkuMjkwOTcgMzIuMzM1NCA5LjEzNTIxQzI4LjY0NDMgOC45Mjg4OCAyNC44Mjk1IDkuNzIzMTggMjEuMzMzNiAxMS40MTI5QzIwLjkxMjMgMTEuNTkwMSAyMC41Mzc2IDExLjgxMDEgMjAuMTcyMiAxMi4wMjQ5TDIwLjAxMDggMTIuMTE3OUMxOS44Nzc0IDEyLjE5NTEgMTkuNzQ0MSAxMi4yNzI0IDE5LjYwOCAxMi4zNTM2QzE5LjMyNTMgMTIuNTE0NiAxOS4wNDkyIDEyLjY3NDQgMTguNzU0NCAxMi44NzkyQzE4LjU0NjMgMTMuMDMyOSAxOC4zMzk1IDEzLjE3NTkgMTguMTMwMSAxMy4zMjNDMTcuNTY1OCAxMy43MjA4IDE2Ljk4NjggMTQuMTMxNyAxNi40OTgzIDE0LjU5NzlDMTQuODQ3NiAxNS45NTI0IDEzLjU1NzEgMTcuNjA3NSAxMi42MDcxIDE4LjkyMTRDMTAuNDM2NSAyMi4xNTY2IDkuMDg2MjIgMjUuOTU2NyA4LjgwNzAyIDI5LjYxNDNMOC43NzY0IDMwLjE1ODhDOC43MzMyOCAzMC45MTk2IDguNjg0NzYgMzEuNzA1NyA4Ljc1MzUzIDMyLjQ1NTVDOC43NjY0OCAzMi42MDg0IDguNzY2MSAzMi43NjM4IDguNzc1MDYgMzIuOTE0QzguNzg4OTUgMzMuMjI5IDguODAxNTIgMzMuNTM3MyA4Ljg0NiAzMy44NjcyTDkuMDczOTYgMzUuNDIyMUM5LjA5NzU2IDM1LjU3NjQgOS4xMTk4IDM1Ljc0MTMgOS4xNjMzIDM1LjkyNjNMOS42NTkxOSAzNy45MjcyTDEwLjEzOCAzOS4yODIzQzEwLjI3MjkgMzkuNjY3MyAxMC40MTU4IDQwLjA3NTEgMTAuNiA0MC40M0MxMi4wMjkyIDQzLjYzNyAxNC4xNDI1IDQ2LjQ1NzggMTYuNzA2MyA0OC41ODVDMTkuMDUwOCA1MC41Mjk2IDIxLjgyNCA1Mi4wMDIzIDI0Ljc0OTEgNTIuODQ1MkwyNi4yMzcxIDUzLjIzNzZDMjYuMzc4MSA1My4yNjkzIDI2LjQ5MjYgNTMuMjg4OSAyNi42MDMxIDUzLjMwNThMMjYuNzc3NSA1My4zMzExQzI3LjAwNTIgNTMuMzYzNiAyNy4yMTk1IDUzLjM5ODYgMjcuNDQ0NSA1My40MzVDMjcuODU5OCA1My41MDc2IDI4LjI2NzIgNTMuNTc0OCAyOC43MDc5IDUzLjYxODNMMzAuNTY0MSA1My43MjI5QzMwLjk1MTYgNTMuNzI0OSAzMS4zMzUyIDUzLjcwNjggMzEuNzA4MSA1My42ODc0QzMxLjkwMzkgNTMuNjgxIDMyLjA5ODQgNTMuNjY4MSAzMi4zMjg4IDUzLjY2MkMzNC41MjUzIDUzLjQ3NzIgMzYuNTEwNiA1My4wNjM0IDM4LjA1MTYgNTIuNDY1MkMzOC4xNzY5IDUyLjQxNzEgMzguMzAwOCA1Mi4zNzk2IDM4LjQyMzQgNTIuMzM1NUMzOC42NzI3IDUyLjI0OTkgMzguOTI1OSA1Mi4xNjcgMzkuMTQzMiA1Mi4wNTk5TDQwLjg1OTEgNTEuMjYyNkw0Mi41NzAyIDUwLjI2NkM0Mi45MDA5IDUwLjA2ODIgNDMuMDIwNSA0OS42NDE0IDQyLjgyODIgNDkuMjk4NEM0Mi42MzIgNDguOTUyNiA0Mi4yMDM0IDQ4LjgzMDggNDEuODYzNCA0OS4wMTY2TDQwLjE3OTIgNDkuOTIxOEwzOC40OTk1IDUwLjYyMjRDMzguMzE2OSA1MC42OTUzIDM4LjEyMSA1MC43NTM0IDM3LjkyMjQgNTAuODE1NUMzNy43ODM4IDUwLjg0ODkgMzcuNjUxOCA1MC44OTgzIDM3LjUwMTIgNTAuOTQwOEMzNi4wNzExIDUxLjQzNSAzNC4yNDQ1IDUxLjc0MjUgMzIuMjQ0IDUxLjgzNDZDMzIuMDQ0MiA1MS44MzgzIDMxLjg0NzEgNTEuODM3OSAzMS42NTQgNTEuODQwM0MzMS4zMDUxIDUxLjg0MTQgMzAuOTYwMiA1MS44NDUxIDMwLjYzOTIgNTEuODMwNUwyOC45MTc3IDUxLjY3MjVDMjguNTQ3NiA1MS42MTkgMjguMTY5NSA1MS41NDI3IDI3Ljc4NDggNTEuNDY3OEMyNy41NjM5IDUxLjQxNjcgMjcuMzM3NiA1MS4zNzM3IDI3LjEyOTkgNTEuMzM3NEwyNi45NTI5IDUxLjI5ODdDMjYuODcwNCA1MS4yODM0IDI2Ljc3NzIgNTEuMjY2NyAyNi43MzMzIDUxLjI1NDNMMjUuMzQ2NiA1MC44MzIyQzIyLjc2NTEgNDkuOTc4OSAyMC4zMyA0OC41NzI5IDE4LjI5NDIgNDYuNzU1N0MxNi4xMDU2IDQ0Ljc5NTEgMTQuMzMzOSA0Mi4yMzM1IDEzLjE3NDIgMzkuMzU4MkMxMi4wMjc2IDM2LjYwMTMgMTEuNTk4OCAzMy4yNzkyIDExLjk3MTYgMzAuMDA3NkMxMi4zMTQ1IDI3LjAyMTMgMTMuMzk0OCAyNC4xNjM1IDE1LjE4NTggMjEuNTA4M0MxNS4zMDM0IDIxLjMzMzkgMTUuNDIxIDIxLjE1OTYgMTUuNTIxMiAyMS4wMTk2QzE2LjQzMDkgMTkuODY4OCAxNy41NDA4IDE4LjU1ODkgMTguOTQ4MyAxNy40OTZDMTkuMzM2NyAxNy4xNTI1IDE5Ljc4NjIgMTYuODU2IDIwLjI2MTEgMTYuNTQ3OEMyMC40ODc4IDE2LjQwMDkgMjAuNzA3OSAxNi4yNTUzIDIwLjg5MDcgMTYuMTMwNkMyMS4wOTc0IDE2LjAwNDggMjEuMzE4OCAxNS44ODMxIDIxLjUzNDggMTUuNzY5NEMyMS42NzYxIDE1LjY5NzUgMjEuODE2MiAxNS42MTkgMjEuOTM4OCAxNS41NTc2TDIyLjEwMDIgMTUuNDY0NkMyMi40MDAyIDE1LjMwMzcgMjIuNjc0OSAxNS4xNTQ2IDIyLjk5MDggMTUuMDM5TDI0LjExODYgMTQuNTcxNUMyNC4zMzk5IDE0LjQ4NDQgMjQuNTcxOCAxNC40MTU5IDI0Ljc5OTcgMTQuMzQ0N0MyNC45NTMgMTQuMjk4MiAyNS4wOTgyIDE0LjI2MzUgMjUuMjYzNSAxNC4yMDc4QzI1Ljc4NiAxNC4wMTgyIDI2LjMyODMgMTMuOTExMiAyNi45MTA1IDEzLjc5NjVDMjcuMTE3IDEzLjc1NzEgMjcuMzMwMiAxMy43MTYzIDI3LjU2MDggMTMuNjU4NUMyNy43NTUzIDEzLjYxMSAyNy45NzM3IDEzLjU5NjkgMjguMjA4MiAxMy41NzYyQzI4LjM2NCAxMy41NjAzIDI4LjUxNzIgMTMuNTQ4MyAyOC42MzE4IDEzLjUzMzNDMjguNzg3NiAxMy41MTczIDI4LjkzNDIgMTMuNTA2NiAyOS4wOTI3IDEzLjQ4NjdDMjkuMzI4NSAxMy40NTU1IDI5LjU0NTYgMTMuNDM0NyAyOS43NDk0IDEzLjQzMzdDMzAuMDIzNyAxMy40NCAzMC4yOTk0IDEzLjQzNTcgMzAuNTc3NyAxMy40Mjc0QzMxLjA4MTEgMTMuNDIxIDMxLjU1NzkgMTMuNDE5NyAzMi4wMzE4IDEzLjQ5MTRDMzQuOTY2NCAxMy43MzUyIDM3LjcxNDQgMTQuNjA4NSA0MC4yMDUyIDE2LjA4NjhDNDIuMzQ4OSAxNy4zNjU1IDQ0LjI3MTYgMTkuMTUyNSA0NS43NjA3IDIxLjI2NEM0Ny4wMjU1IDIzLjA2MjggNDcuOTc1NiAyNS4wNTI4IDQ4LjQ5MjggMjcuMDM5M0M0OC41NzIgMjcuMzE3NiA0OC42Mjk5IDI3LjU5MzEgNDguNjgzOSAyNy44NjU5QzQ4LjcxNTQgMjguMDQyOCA0OC43NTYzIDI4LjIxNDUgNDguNzg5MiAyOC4zNjM2QzQ4LjgwMzcgMjguNDU0MSA0OC44MjA4IDI4LjU0MDYgNDguODQ0NSAyOC42MjU4QzQ4Ljg3NDkgMjguNzQ0MyA0OC44OTg2IDI4Ljg2NCA0OC45MTE2IDI4Ljk2NTFMNDguOTc5MyAyOS42MDQ3QzQ4Ljk5MjIgMjkuNzc0OCA0OS4wMTMyIDI5LjkzMzEgNDkuMDMwMSAzMC4wODg3QzQ5LjA2NjggMzAuMzI2OCA0OS4wODg5IDMwLjU2MDggNDkuMDk2NCAzMC43NTYxTDQ5LjEwODMgMzEuOTAwMUM0OS4xMzEyIDMyLjMzMDcgNDkuMDg5IDMyLjcxMTYgNDkuMDUyMiAzMy4wNjczQzQ5LjAzODQgMzMuMjU5OCA0OS4wMTI2IDMzLjQ0NDMgNDkuMDEyMyAzMy41ODI0QzQ4Ljk5NjEgMzMuNjkyNiA0OC45OTE4IDMzLjc5MzUgNDguOTgzNiAzMy44OTE3QzQ4Ljk3NTMgMzQuMDA3MiA0OC45NzI0IDM0LjExNDggNDguOTQxNCAzNC4yNTU0TDQ4LjU0NDkgMzYuMzA1OUM0OC4zMTM0IDM3Ljg2MjMgNDkuMzc5MyAzOS4zMzY1IDUwLjk0ODggMzkuNTgyMkM1Mi4wNDE3IDM5Ljc2MDEgNTMuMTUzNiAzOS4yODE5IDUzLjc3MTEgMzguMzY2NEM1NC4wMDYzIDM4LjAxNzYgNTQuMTYwNCAzNy42MjU3IDU0LjIyMjcgMzcuMjA2NEw1NC41MjE3IDM1LjI1NzRDNTQuNTUxNCAzNS4wNzU2IDU0LjU3MiAzNC44MyA1NC41ODQ2IDM0LjU3OTFMNTQuNjAyOCAzNC4yMzM4QzU0LjYwOTggMzQuMDU5OCA1NC42MjIzIDMzLjg3NzkgNTQuNjM0NyAzMy42Nzg4QzU0LjY3MzQgMzMuMTA1MiA1NC43MTYzIDMyLjQ0NzkgNTQuNjYxOSAzMS44MDU4TDU0LjU4NjcgMzAuNDI4OUM1NC41NjIyIDMwLjA5NTIgNTQuNTA5NyAyOS43NiA1NC40NTU5IDI5LjQxODFDNTQuNDMxIDI5LjI1NzIgNTQuNDA0OCAyOS4wODk2IDU0LjM4MjYgMjguOTA3NEw1NC4yNjg3IDI4LjEwNEM1NC4yMzMyIDI3LjkyNDQgNTQuMTgwNCAyNy43MjczIDU0LjEzMjkgMjcuNTM5Nkw1NC4wNjQzIDI3LjI0NTRDNTQuMDE5NSAyNy4wNzEgNTMuOTc3MyAyNi44OTI3IDUzLjkzMzggMjYuNzA3NkM1My44NDU1IDI2LjMzMDkgNTMuNzQ3OSAyNS45NDIyIDUzLjYxMyAyNS41NTcxQzUyLjg0IDIzLjAyOTIgNTEuNTM4MyAyMC41MTk0IDQ5LjgzMzggMTguMjc5OUM0Ny44NTQ0IDE1LjY4MiA0NS4zMzMzIDEzLjUwODcgNDIuNTU2MyAxMS45ODE2WiIgZmlsbD0iIzQ5NDVGRiIvPgo8L3N2Zz4K"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
  </main>
  <div
    class="c37"
  >
    <p
      aria-live="polite"
      aria-relevant="all"
      id="live-region-log"
      role="log"
    />
    <p
      aria-live="polite"
      aria-relevant="all"
      id="live-region-status"
      role="status"
    />
    <p
      aria-live="assertive"
      aria-relevant="all"
      id="live-region-alert"
      role="alert"
    />
  </div>
</div>
`;
